### ISSUE TITLE
Does UnrolledList support work-stealing?

### ISSUE THREAD

**Issue by zoujiaqing** [ikod/ikod-containers#7]
I am thinking of improving the task distribution performance of worker threads, hopefully in the form of work stealing.

**Comment by ikod** [comment]
> I am thinking of improving the task distribution performance of worker threads, hopefully in the form of work stealing.

Unrolled list is just a list. I'm not sure if I understand your question.

**Comment by zoujiaqing** [comment]
Thanks, Is it thread-safe?

**Comment by ikod** [comment]
No, it is not thread-safe. It can be wrapped in rw-locks if this fits your
requirements.

вт, 24 мая 2022 г., 16:46 zoujiaqing ***@***.***>:
